src/target/riscv/riscv.c

```diff
@@ -3590,6 +3817,20 @@ static const struct command_registration riscv_exec_command_handlers[] = {
 		.help = "Control dcsr.ebreaku. When off, U-mode ebreak instructions "
 			"don't trap to OpenOCD. Defaults to on."
 	},
+	{
+		.name = "etrigger",
+		.handler = riscv_etrigger,
+		.mode = COMMAND_ANY,
+		.usage = "set [vs] [vu] [nmi] [m] [s] [u] <mie bits>|clear",
+		.help = "Set or clear a single interrupt trigger."
```

Comment:
There seem to be few copy-paste mistakes:

```suggestion
		.name = "etrigger",
		.handler = riscv_etrigger,
		.mode = COMMAND_ANY,
		.usage = "set [vs] [vu] [m] [s] [u] <exception_codes>|clear",
		.help = "Set or clear a single exception trigger."
```

Reply:
Thank you.